sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -584,7 +584,7 @@ export interface CreateOperationInput {
     // (undocumented)
     operationType: typeof BulkOperationType.Create;
     // (undocumented)
-    partitionKey?: string | number | null | Record<string, unknown> | undefined;
```

Comment:
This would also be a breaking change. Previously users would be able to set PartitionKey as any object. Now PartitionKey won't allow that

Reply:
So this was to allow for `{}` `partitionKey`, which is compliant with newer `partitionKey` definition.